api.md

```diff
@@ -580,11 +580,33 @@ the same semantics as the `show_chunks` [function](#show_chunks).
 
 |Name|Description|
 |---|---|
-| `table_name` | Hypertable name from which to drop chunks. If not supplied, all hypertables are affected.
+| `table_name` | Hypertable name or continuous aggregate view from which to drop chunks. This is optional, but see below.
 | `schema_name` | Schema name of the hypertable from which to drop chunks. Defaults to `public`.
 | `cascade` | Boolean on whether to `CASCADE` the drop on chunks, therefore removing dependent objects on chunks to be removed. Defaults to `FALSE`.
 | `cascade_to_materializations` | Set to `TRUE` to also remove chunk data from any associated continuous aggregates. Set to `FALSE` to only drop raw chunks (while keeping data in the continuous aggregates). Defaults to `NULL`, which errors if continuous aggregates exist.|
 
+The `table_name` is optional and can be used to remove chunks from all
```

Comment:
```suggestion

```
Please remove the line for reasons stated above. Would rather not have users trying this out without table_name.

Reply:
I added a tip that omitting the table name is deprecated and that the table name should be given.